### PR TITLE
add limit to creation of plotNFT's from the CLI

### PR DESCRIPTION
**Pull request by jack60612**
limit is 18 and can be overridden with --override_limit

### CODE REVIEW

File: chia/cmds/plotnft_funcs.py [hunk 1]
```diff
@@ -56,6 +56,21 @@ async def create(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -
...
+    farmer_rpc_port = config["farmer"]["rpc_port"]
+    # get total amount of plotnft's
+    farmer_client = await FarmerRpcClient.create(self_hostname, uint16(farmer_rpc_port), DEFAULT_ROOT_PATH, config)
+    total_plot_nfts = len((await farmer_client.get_pool_state())["pool_state"])
+    farmer_client.close()
+    await farmer_client.await_closed()
+
+    if total_plot_nfts >= 20 and not override_limit:
+        raise Exception(
+            "20 PlotNFT's Already exist. "
```
Comment (mariano54) [inline chia/cmds/plotnft_funcs.py:71]: Change Already to already

File: chia/cmds/plotnft_funcs.py [hunk 2]
```diff
@@ -56,6 +56,21 @@ async def create(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -
...
+            "If you need to override this limit (Most users should not) use  '--override_limit'."
```
Comment (mariano54) [inline chia/cmds/plotnft_funcs.py:72]: Change Most to most

File: chia/cmds/plotnft_funcs.py (hunk 1 above)
Comment (jack60612) [reply chia/cmds/plotnft_funcs.py]: Damn english lint.

**Comment by jack60612** [comment]
Awaiting Testing

**Comment by jack60612** [comment]
oops, forgot to add earlier, tests passed successfully @mariano54

**Review by mariano54 — APPROVED** [review]

**Pull request merged**